x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/description_step/helpers.tsx

```diff
@@ -490,20 +490,29 @@ export const buildRuleTypeDescription = (label: string, ruleType: Type): ListIte
   }
 };
 
-export const buildThresholdDescription = (label: string, threshold: Threshold): ListItems[] => [
-  {
-    title: label,
-    description: (
-      <>
-        {isEmpty(threshold.field[0])
-          ? `${i18n.THRESHOLD_RESULTS_ALL} >= ${threshold.value}`
-          : `${i18n.THRESHOLD_RESULTS_AGGREGATED_BY} ${
-              Array.isArray(threshold.field) ? threshold.field.join(',') : threshold.field
-            } >= ${threshold.value}`}
-      </>
-    ),
-  },
-];
+export const buildThresholdDescription = (
+  label: string,
+  threshold: FieldValueThreshold
+): ListItems[] => {
+  let thresholdDescription = isEmpty(threshold.field[0])
+    ? `${i18n.THRESHOLD_RESULTS_ALL} >= ${threshold.value}`
+    : `${i18n.THRESHOLD_RESULTS_AGGREGATED_BY} ${
+        Array.isArray(threshold.field) ? threshold.field.join(',') : threshold.field
```

Comment:
It seems we don't need this `isArray` check since `threshold.field` will always be an array.

Reply:
I agree. It's not part of my changes (this was the existing logic), but I will improve it. 
Let's hope that it really always is an array, because from what I saw about how some typescript things work here, we can never be sure :-)

Reply:
Yeah, typing is not very robust on this page. But the change is safe. `field` is converted to an array [here](https://github.com/jkelas/kibana/blob/b103ba6bd16ff6011ca74b7ba85ad261f63d85d8/x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/helpers.tsx#L139), before we pass this data into `buildThresholdDescription`.